api/v1alpha3/azurecluster_types.go

```diff
@@ -50,6 +50,8 @@ type AzureClusterSpec struct {
 type AzureClusterStatus struct {
 	Network Network `json:"network,omitempty"`
 
+	FailureDomains clusterv1.FailureDomains `json:"failureDomains,omitempty"`
```

Comment:
There should probably also be a conversion in place for AzureCluster.Status to avoid lossy conversions.

Reply:
I will add this conversion 👍

Reply:
Just added that